analyzer-models/src/main/java/ffb/analyzer/models/espn/deserializers/ScoresByStatDeserializer.java

```diff
@@ -0,0 +1,44 @@
+package ffb.analyzer.models.espn.deserializers;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonNode;
+import ffb.analyzer.models.espn.ScoreByStat;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map.Entry;
+
+/**
+ * Deserializer to turn JSON into a list of {@link ScoreByStat}.
+ */
+public class ScoresByStatDeserializer extends BaseObjectDeserializer<List<ScoreByStat>> {
+    private static final String INELIGIBLE_FIELD_NAME = "ineligible";
+    private static final String RANK_FIELD_NAME = "rank";
+    private static final String SCORE_FIELD_NAME = "score";
+
+    @Override
+    public List<ScoreByStat> deserialize(JsonParser jsonParser, DeserializationContext deserializationContext)
+        throws IOException
+    {
+        JsonNode node = jsonParser.getCodec().readTree(jsonParser);
+        Iterator<Entry<String, JsonNode>> fields = node.fields();
+        List<ScoreByStat> scores = new ArrayList<>();
+
+        fields.forEachRemaining(field -> {
```

Comment:
We could `fields.toList().stream()...` to collect these into a list instead

Reply:
The full chain would be 
``` node.getFields()
  .stream()
  .map(field -> ... return score)
  .collect(Collectors.toList();
```

So it's a cursor right up until it's list.

Reply:
Never mind. Not sure where I found `toList`, but it's not on a `Iterator`, so we're keeping this as is.